racingcar-java/src/main/java/RacingCar.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RacingCar {
+
+    List<Car> cars;
```

Comment:
```suggestion
    private final List<Car> cars;
```
일급 컬랙션을 사용하셨네요.
다만 일급 컬랙션을 사용하는 이유는 컬랙션 객체에 대해서 사용하는 부분에 대한 접근을 최소화하고, 그렇게 감싼 일급 컬랙션 값에 대해서 필요한 부분을 요청하게 함으로써 객체처럼 적절한 역할을 부여하는데 있어요.
그렇게하면 차후에 이 코드를 보거나 다른사람이 코드를 봐도 컬랙션을 어떻게 사용했는지 명시적으로 바로 알 수 있으니까요.

이렇게 필드값을 그대로 노출하면 `racingCar.cars.removeAll()`이라는 요청에 대해서 같은 패키지 내에서는 접근제어자가 접근이 가능하겠죠.

또 일급컬랙션에 대해서 여러가지 장점이 있으니 한번 알아보시는 것도 좋을 것 같네요 :)
[https://jojoldu.tistory.com/412](url)